app/views/unattended/provisioning_templates/snippet/built.erb

```diff
@@ -45,7 +46,15 @@ elif [ -x /usr/bin/wget ]; then
 else
   wget -q -O /dev/null --header 'Content-Type: text/plain' '<%= url %>'
 fi
+FINAL_STATUS=$?
 
 if [ -x "$(command -v subscription-manager)" ] ; then
   subscription-manager facts --update
+  SUB_MAN_STATUS=$?
+
+  if [ $FINAL_STATUS -eq 0 ]; then
+    FINAL_STATUS=$SUB_MAN_STATUS
+  fi
 fi
+
+(exit $FINAL_STATUS);
```

Comment:
The parenthesis here will ensure it exits with the correct code, and then when the `host_init_config_default` script continues `$?` will be correct.

Reply:
It opens a subshell, so it makes sense but I wonder what the point is

Reply:
So we launch a sub shell only to have it exit with a predefined exit code? Clever